app/models/concerns/essi/dynamic_solr_document.rb

```diff
@@ -17,13 +17,21 @@ def load_allinson_flex
         # profile directly.
         profile = AllinsonFlex::Profile.current_version
```

Comment:
Would it be beneficial to eager load .properties here?

Reply:
We could, but this code doesn't load property associations in the way DynamicCatalogBehavior does. It should only ever need 3 sql queries at most and that includes the count in the log string.